src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -0,0 +1,27 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumbers {
+
+    private List<Integer> numbers;
+
+    public LottoNumbers(int ... numbers) {
+        this.numbers = IntStream.of(numbers).boxed().collect(Collectors.toList());
```

Comment:
method chaining을 하실때는 행 구분을 해주시는것이 좋습니다.

```suggestion
        this.numbers = IntStream.of(numbers)
                                .boxed()
                                .collect(Collectors.toList());
```